### PR TITLE
Integrate X3D JSON format into loader.  Does not depend on DOM.

### PR DESCRIPTION
**Pull request by coderextreme**
see issue

### CODE REVIEW

**Comment by coderextreme** [comment]
I tried to make it so it would still work as part of a project.  I am not sure how well I succeeded.

**Comment by jonaskello** [comment]
This looks interesting :-). So the goal is to add support for x3d json encoding as described [here](http://www.web3d.org/wiki/index.php/X3D_JSON_Encoding)?

File: package.json [hunk 1]
```diff
@@ -24,10 +24,13 @@
     "webpack-dev-server": "^1.16.2"
   },
   "dependencies": {
+    "express": "^4.15.2",
```
Comment (jonaskello) [inline package.json:27]: This should be a devDependency since express is not required to use the loader package.

**Comment by coderextreme** [comment]
Yes, that is the correct spot for the standardizing effort.   There is a
whole bunch of json files you can download from the X3D resources examples
site.

Best, John

**Comment by coderextreme** [comment]
Well, in order to run the new app.js, I require express.   If it's still a
dev dependency, or you have a better way of running a web server to test,
let me know (instructions?).

**Pull request closed without merging**